src/local_scheduler/local_scheduler_algorithm.h

```diff
@@ -307,6 +304,65 @@ int reconstruct_object_timeout_handler(event_loop *loop,
 void print_worker_info(const char *message,
                        SchedulingAlgorithmState *algorithm_state);
 
+/**
+ * Get the number of tasks, per actor handle, that have been executed on an
+ * actor so far.
+ *
+ * @param algorithm_state State maintained by the scheduling algorithm.
+ * @param actor_id The ID of the actor whose task counters are returned.
+ * @return A map from handle ID to the number of tasks submitted by that handle
+ *         that have executed so far.
+ */
+std::unordered_map<ActorID, int64_t, UniqueIDHasher> get_actor_task_counters(
+    SchedulingAlgorithmState *algorithm_state,
+    ActorID actor_id);
+
+/**
+ * Set the number of tasks, per actor handle, that have been executed on an
+ * actor so far. All previous counts will be overwritten.
+ *
+ * @param algorithm_state State maintained by the scheduling algorithm.
+ * @param actor_id The ID of the actor whose task counters are returned.
+ * @param task_counters A map from handle ID to the number of tasks submitted
+ *        by that handle that have executed so far.
+ * @return Void.
+ */
+void set_actor_task_counters(
+    SchedulingAlgorithmState *algorithm_state,
+    ActorID actor_id,
```

Comment:
const ref, same comment applies in a bunch of places

Reply:
+1; const -> happiness